newbius/installations/example/main.tf

```diff
@@ -20,6 +20,17 @@ module "filestore" {
     } : null
   }
 
+  accounting_storage = {
```

Comment:
This field is nullable and you use it to resolve whether we need to create filestores for accounting. Going this way, won't it just fail because it's not null, but both `spec` and `existing` are not specified in case we don't want to use accounting.
I think it must be passed only if `var.accounting_enabled` is true. Otherwise, `null`